libs/core/src/lib/toggle/toggle.component.spec.ts

```diff
@@ -84,4 +84,16 @@ describe('ToggleComponent', () => {
         expect(toggle.classList).toContain('fd-toggle--' + size)
```

Comment:
I don't know what are the guidlines but this can be writen as `fd-toggle--${size}`. Typescript will transpile it to js